lighthouse-core/report/html/report-styles.css

```diff
@@ -548,11 +551,66 @@
 }
 
 .lh-metric__description {
+  display: none;
   color: var(--secondary-text-color);
+  padding: var(--expandable-padding);
+}
+.lh-audit-group--metrics__show-descriptions .lh-metric__description {
+  display: block;
 }
 
 .lh-metric__value {
   white-space: nowrap; /* No wrapping between metric value and the icon */
+  font-weight: 500;
+}
+
+.lh-metrics-toggle {
+  position: relative;
+}
+.lh-metrics-toggle input {
+  cursor: pointer;
+  left: 0;
+  opacity: 0;
+  position: absolute;
+  top: 0;
+  width: 74px;
```

Comment:
instead of these magic numbers you could just do bottom:0; right: 0; and now its 100% 100%. ;)

also not putting left & top together in a rule? what kind of sadist are you?  😭